posthog/async_migrations/test/test_0006_persons_and_groups_on_events_backfill.py

```diff
@@ -262,7 +285,7 @@ def test_data_copy_groups(self):
                 "group0_properties": json.dumps({"industry": "IT"}),
                 "group0_created_at": "2022-01-02T00:00:00Z",
                 "$group_1": "77",
-                "group1_properties": "",
+                "group1_properties": "{}",
```

Comment:
context: the backfill now sets this to `{}` if there's no corresponding group. This allows us to easily spot columns for which the backfill didn't work vs. columns for which it worked but the group didn't exist. Also helps on the queries side